xbmc/dbwrappers/sqlitedataset.cpp

```diff
@@ -25,6 +25,7 @@
 
 namespace {
 #define X(VAL) std::make_pair(VAL, #VAL)
+//!@todo Remove ifdefs when sqlite version requirement has been bumped
```

Comment:
Perhaps add which version is expected

Reply:
That means I have to crawl through all of them but yeah I guess you are right :o)

Reply:
Well at least some rough guestimate. Compiler errors will show again if still not the correct version